navbar.html

```diff
@@ -10,11 +14,17 @@
         <a class="nav-link" href="https://engrevue.co.nz/">Home</a>
       </li>
       <li class="nav-item">
-        <a class="nav-link" href="https://engrevue.co.nz/previous.html">Previous Revues</a>
+        <a class="nav-link" href="https://engrevue.co.nz/previous">Previous Revues</a>
```

Comment:
One possible change we could do to make this even tidier would just be relative links? 

I.e. i think just `/previous` would work too.